pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -18,6 +18,19 @@ def load_earth_magnetic_anomaly(
     r"""
     Load the Earth magnetic anomaly datasets in various resolutions.
 
+    .. list-table::
+       :widths: 50 50
+       :header-rows: 1
+
+       * - Global Earth Magnetic Anomaly Model (EMAG2)
+         - World Digital Magnetic Anomaly Map (WDMAM)
+       * - .. figure::
+              https://www.generic-mapping-tools.org/
+              remote-datasets/_images/GMT_mag.jpg
+         - .. figure::
+              https://www.generic-mapping-tools.org/
+              remote-datasets/_images/GMT_wdmam.jpg
```

Comment:
Does this work?
```suggestion
       * - .. figure::
              https://www.generic-mapping-tools.org/remote-datasets/_images/GMT_mag.jpg # noqa: W505
         - .. figure::
              https://www.generic-mapping-tools.org/remote-datasets/_images/GMT_wdmam.jpg # noqa: W505
```

Reply:
Locally, the Style Checks fail (even when `#noqa w505` is used):
```
************* Module pygmt.datasets.earth_magnetic_anomaly
pygmt/datasets/earth_magnetic_anomaly.py:30:0: C0301: Line too long (102/100) (line-too-long)
```

Reply:
What's your `flakeheaven` and `pylint` versions? 

I have 
```
$ flakeheaven --version
FlakeHeaven 3.3.0
Flake8    4.0.1

$ pylint --version
pylint 3.0.1
astroid 3.0.0
Python 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:40:35) [GCC 12.3.0]
```
and it seems working for me.